src/main/java/com/google/devtools/build/lib/rules/repository/RepositoryFunction.java

```diff
@@ -581,8 +582,7 @@ public static void addExternalFilesDependencies(
       if (isDirectory || repositoryPath.segmentCount() > 1) {
         if (!isDirectory
             && rule.getRuleClass().equals(LocalRepositoryRule.NAME)
-            && (repositoryPath.endsWith(BuildFileName.WORKSPACE.getFilenameFragment())
-                || repositoryPath.endsWith(BuildFileName.WORKSPACE_DOT_BAZEL.getFilenameFragment()))) {
+            && WorkspaceFileHelper.endsWithWorkspaceFileName(repositoryPath)) {
```

Comment:
Same here -- we are in a SkyFunction, you must ask Skyframe about file existence, you shouldn't rely on Path.exists()

Reply:
Ah sorry, this is not about existence check, merely checking the file name. This is fine.

Reply:
No problem, thanks!